frontend/catalyst/debug/__init__.py

```diff
@@ -38,4 +39,5 @@
     "instrumentation",
     "replace_ir",
     "compile_executable",
+    "is_debugger_active",
```

Comment:
This module exposes the user facing API (the stuff that renders in the docs), do we expect this function is something a user would use or an internal function?

Reply:
Ah, good point yea. I would say we can keep this internal.

Reply:
Done